src/ontology/uberon-edit.obo

```diff
@@ -928,7 +928,7 @@ intersection_of: has_quality PATO:0002236 ! aliform
 relationship: capable_of_part_of GO:0060361 ! flight
 relationship: in_taxon NCBITaxon:33213 ! Bilateria
 relationship: never_in_taxon NCBITaxon:118072 {source="bgee"} ! Coelacanthimorpha
-relationship: never_in_taxon NCBITaxon:186634 {source="bgee", seeAlso="Wikipedia:Flying_fish"} ! Otomorpha
+relationship: never_in_taxon NCBITaxon:186634 {seeAlso="Wikipedia:Flying_fish", source="bgee"} ! Otomorpha
```

Comment:
Can you check that you are using the latest version of Protégé (5.6.5)?

Those changes (caused by the fact that the way qualifiers are sorted when being written has changed in recent OWLAPI versions) should not be here.

Reply:
oh shoot, I am not. If I re-download it will it fix the issues?